compose/ui/ui/src/androidMain/kotlin/androidx/compose/ui/window/AndroidPopup.android.kt

```diff
@@ -219,10 +98,10 @@ fun Popup(
  * @param content The content to be displayed inside the popup.
  */
 @Composable
-fun Popup(
+actual fun Popup(
     popupPositionProvider: PopupPositionProvider,
-    onDismissRequest: (() -> Unit)? = null,
-    properties: PopupProperties = PopupProperties(),
```

Comment:
When we change default arguments in simple Kotlin functions, it will break binary backward compatibility.
But I don't know how it will work with expect / actual function with same default arguments.

@igordmn Do you know how it will behave with backward compatibility?

Reply:
It doesn't. You cannot add default values to actual functions, only to expect. There is no change and choice here

Reply:
I checked with JVM - expect / actual default arguments don't break binary backward compatibility.